mlir/test/Analysis/DataFlow/test-dead-code-analysis-regression.mlir

```diff
@@ -0,0 +1,5 @@
+// RUN: mlir-opt -int-range-optimizations %s 
+
+%0 = scf.execute_region -> tensor<5x16xi16> {
```

Comment:
suggestion: please merge this test case with one of the existing ones for dataflow analysis.

Also, this seems like invalid IR. `scf.execute_region` should be verifying its terminators